src/plugin.ts

```diff
@@ -68,6 +68,12 @@ const EXPORT_NAME = 'openGraphImage'
 const CACHE_FILE = 'node_modules/.vite/cache/remix-og-image/cache.json'
 
 export function openGraphImagePlugin(options: Options): Plugin {
+  if (path.isAbsolute(options.outputDirectory)) {
```

Comment:
Providing an absolute path as `outputDirectory` is forbidden. The path must always be relative.